assets/src/ba_data/python/bastd/game/assault.py

```diff
@@ -176,7 +176,7 @@ def _handle_base_collide(self, team: ba.Team) -> None:
             return
 
         # If its another team's player, they scored.
-        player_team = player.get_team()
+        player_team = player.team
```

Comment:
Don't worry about it for now, but in the future could I ask that you fix the type-checking in cases like this in addition to the error? Mypy doesn't know what player_team is here, otherwise it would have raised an alarm for the incorrect get_team() call.

Basically, if you temporarily add a `reveal_type(player)` statement and then run `make mypy`, you'll probably see that player is considered `Any`. If you add an `assert isinstance(player, ba.Player)` right after player is assigned, this will tell the type checker that it is in fact a Player and it'll do better at catching errors (both this one and others in the future that may arise)

See info here: https://github.com/efroemling/ballistica/wiki/Knowledge-Nuggets#beware-the-any-type

Reply:
Thanks I got it. I will try to use this in future.